Source/PlayerEx.cpp

```diff
@@ -101,25 +116,103 @@ int PlayerEx::GetSatelliteCount(map_id objectType)
 		return playerArray[playerNum].satellites.solarSat;
 	case mapEDWARDSatellite:
 		return playerArray[playerNum].satellites.edwardSat;
+	case mapEvacuationModule:
+		return playerArray[playerNum].satellites.evacModule;
+	case mapChildrenModule:
+		return playerArray[playerNum].satellites.childCargo;
+	case mapRareMetalsCargo:
+		return playerArray[playerNum].satellites.rareCargo;
+	case mapCommonMetalsCargo:
+		return playerArray[playerNum].satellites.commonCargo;
+	case mapFoodCargo:
+		return playerArray[playerNum].satellites.foodCargo;
+	case mapPhoenixModule:
+		return playerArray[playerNum].satellites.phoenixModule;
+	case mapOrbitalPackage:
+		return playerArray[playerNum].satellites.orbitalPackage;
+	case mapStasisSystems:
+		return playerArray[playerNum].satellites.stasisSystems;
+	case mapSkydock:
+		return playerArray[playerNum].satellites.skydock;
+	case mapSensorPackage:
+		return playerArray[playerNum].satellites.sensorPackage;
+	case mapHabitatRing:
+		return playerArray[playerNum].satellites.habitatRing;
+	case mapFuelingSystems:
+		return playerArray[playerNum].satellites.fuelingSystems;
+	case mapCommandModule:
+		return playerArray[playerNum].satellites.commandModule;
+	case mapFusionDriveModule:
+		return playerArray[playerNum].satellites.fusionDrive;
+	case mapIonDriveModule:
+		return playerArray[playerNum].satellites.ionDrive;
 	default:
 		return 0;
 	}
 }
 
 void PlayerEx::SetSatelliteCount(map_id objectType, int count)
 {
-	if (!isInited) {
+	if (!isInited)
 		return;
-	}

```

Comment:
Ditto.

Reply:
I think this is all set, could you take another look and confirm?  I'm still figuring out how GitHub works so it's possible I've messed something up along the way, and a commit didn't go through, or I didn't do whatever I needed to get the updated code to show up in this review.